tests/test_column.py

```diff
@@ -196,6 +196,71 @@ def test_map(column, map):
     pd.testing.assert_series_equal(result, expected)
 
 
+def test_map_returns_categorical_dtype():
+
+    # use scheme with labels
```

Comment:
Maybe it makes sense to track to which issue this belongs?

```suggestion
    # See https://github.com/audeering/audformat/issues/317
    
    # use scheme with labels
```